server/master/src/main/java/org/apache/accumulo/master/tableOps/tableImport/CreateImportDir.java

```diff
@@ -47,10 +47,9 @@
     UniqueNameAllocator namer = master.getContext().getUniqueNameAllocator();
 
     Path exportDir = new Path(tableInfo.exportDir);
-    String[] tableDirs = ServerConstants.getTablesDirs(master.getContext());
+    Set<String> tableDirs = ServerConstants.getTablesDirs(master.getContext());
 
-    log.info("Looking for matching filesystem for " + exportDir + " from options "
-        + Arrays.toString(tableDirs));
+    log.info("Looking for matching filesystem for " + exportDir + " from options " + tableDirs);
```

Comment:
Could use sl4j parameterized log statement. 
```suggestion
    log.info("Looking for matching filesystem for {} from options {}", exportDir, tableDirs);
```

Reply:
This was preexisting, but good to change while we're in there.